### ISSUE TITLE
Sourcery intermittently throwing error messages in PyCharm

### ISSUE THREAD

**Issue by Hellebore** [sourcery-ai/sourcery#92]
### Issue description or question
Sourcery intermittently throws up 'Unable to connect to Sourcery executable' error in PyCharm.

### Sourcery Version

0.8.1

### Code editor or IDE name and version

Pycharm 2020.2

### OS name and version

Windows
OSX
Linux

**Comment by Tangerino** [comment]
A bit of more details

Got this message today in PyCharm event log

```
10/31/20
08:24	Unable to connect to Sourcery executable

08:24	Failed to authenticate with Sourcery
				502 Server Error
				
				
				Error: Server Error
				The server encountered a temporary error and could not complete your request.
				Please try again in 30 seconds.
```

**Comment by brendanator** [comment]
That looks like a temporary server error. Is this still occurring @Tangerino?

**Comment by Hellebore** [comment]
Potential fix is now live with version 0.8.4 - please re-open if it doesn't address the issue.
